kernel/safe_typing.ml

```diff
@@ -346,10 +346,10 @@ let check_required current_libs needed =
     try
       let actual = DPMap.find id current_libs in
       if not(digest_match ~actual ~required) then
-        CErrors.error
-          ("Inconsistent assumptions over module "^(DirPath.to_string id)^".")
+        CErrors.user_err Pp.(pr_sequence str
```

Comment:
AFAICT yes, why? Here it introduces one extra space indeed so we could use `pr_list` if desired.